src/Sulu/Component/Content/Metadata/PropertiesMetadata.php

```diff
@@ -111,6 +111,19 @@ private function removeSectionProperties(array $children)
                 continue;
             }
 
+            if ($child instanceof BlockMetadata) {
+                // deep cloning is necessary because the components must only be manipulated for these properties
+                $clone = \unserialize(\serialize($child));
```

Comment:
Doesn't [`clone`](https://www.php.net/manual/en/language.oop5.cloning.php) work?

Reply:
Nope because `clone` copies the object but uses the same references for the child objects

Reply:
Why not override the `__clone` as it is shown in the [first example of the clone manual](https://www.php.net/manual/en/language.oop5.cloning.php)? I think that's what I would expect here to happen.

Reply:
Ok I will try it 👍

Reply:
@danrot fixed it